style.css

```diff
@@ -13,7 +13,6 @@ This theme, like WordPress, is licensed under the GPL.
 
 @import url(https://unpkg.com/browse/bulma@0.9.3/);
 @import url(https://unpkg.com/@creativecommons/fonts@2020.9.3/css/fonts.css);
```

Comment:
While we're here, should we go ahead and move these imports to the `header.php` as well? I.e., they're also not needed in the WordPress admin area.

Reply:
I have this warning when the browser tries to load Bulma: 
```
Resource interpreted as Stylesheet but transferred with MIME type text/html: "https://unpkg.com/browse/bulma@0.9.3/".
```

Reply:
can I use this link instead `https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css` as shown on the Bulma website